OpenRA.Mods.Common/Activities/DeliverResources.cs

```diff
@@ -18,6 +18,15 @@ namespace OpenRA.Mods.Common.Activities
 	public class DeliverResources : Activity
 	{
 		const int NextChooseTime = 100;
+
+		IMove movement;
+		Harvester harv;
+		HarvesterInfo harvInfo;
+
+		// TODO: This is a hack because this class is sometimes
+		// constructed by reflection and cannot add an Actor variable in
```

Comment:
I just didn't look up the reason for this approach in ```FindResources``` and wrongly assumed it would apply here, too.